web/src/components/footer/subcomponents/contact/contact.js

```diff
@@ -1,19 +1,31 @@
 import { Box, Link, Typography } from "@mui/material";
 import contactStyles from "./contact.styles";
+import EmailIcon from "@mui/icons-material/Email";
+import FacebookIcon from "@mui/icons-material/Facebook";
+import InstagramIcon from "@mui/icons-material/Instagram";
```

Comment:
We can combine these imports I think, same with DiscountRounded (can we alias that to DiscordIcon?)

Reply:
No I don't think so, since they're all default imoprts

Reply:
and I tried so probably not